src/main/java/frc/robot/commands/basic/autoCmd.java

```diff
@@ -2,7 +2,7 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-package frc.robot.commands;
+package frc.robot.commands.basic;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
```

Comment:
Can you change the name of this file to `AutoCmd.java`  (and update the class to match)

Same with `teleopCmd.java`, `controllerSubsystem.java`, and `drivetrainSubsystem.java`